LayoutTests/fast/forms/ios/color-show-picker.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+<script src="../../../resources/ui-helper.js"></script>
+<style>
+input {
+    width: 300px;
+    height: 50px;
+}
+
+body {
+    margin: 0;
+}
+</style>
+<head>
+<body onload="runTest()">
+<input id="input" type="color"/>
+<pre>Is showing color picker? <span id="before"></span></pre>
+<br>
+<div>This test verifies that a color picker is presented when calling showPicker on a color input.</div>
+</body>
+<script>
+if (window.testRunner) {
+    testRunner.waitUntilDone();
+    testRunner.dumpAsText();
+}
+
+function getFormControlInteractionScript()
+{
+    return `
+    (function() {
+        uiController.didStartFormControlInteractionCallback(function () {
+            uiController.uiScriptComplete();
+        })
+    })();`
+}
+
+async function runTest() {
+    await UIHelper.activateAt(0, 0);
+    input.showPicker();
```

Comment:
This is throwing despite it being almost identical to the other tests? It should definitely have a user gesture from the activate at?